x-pack/plugins/security_solution/server/lib/detection_engine/signals/threat_mapping/types.ts

```diff
@@ -177,14 +180,28 @@ export interface GetSortWithTieBreakerOptions {
   listItemIndex: string;
 }
 
+export interface ThreatListDoc {
+  [key: string]: unknown;
+}
+
 /**
  * This is an ECS document being returned, but the user could return or use non-ecs based
  * documents potentially.
  */
-export interface ThreatListItem {
+export type ThreatListItem = SearchResponse<ThreatListDoc>['hits']['hits'][number];
```

Comment:
@FrankHassanabad @marshallmain I restructured these types as I think we were previously conflating the idea of the document in _source with the object in a response's `hits.hits`. Those are now captured with `ThreatListDoc` and `ThreatListItem`, respectively.